app/src/main/java/org/fossasia/openevent/general/event/subtopic/EventSubTopicConverter.kt

```diff
@@ -0,0 +1,17 @@
+package org.fossasia.openevent.general.event.subtopic
+
+import androidx.room.TypeConverter
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+
+class EventSubTopicConverter {
+    @TypeConverter
+    fun toEventSubTopic(json: String): EventSubTopic? {
+        val objectMapper = ObjectMapper()
```

Comment:
@iamareebjamal updated

Reply:
No, this is still instantiating ObjectMapper yourself

Reply:
@iamareebjamal 
     val mapType = object : TypeReference<EventSubTopic>() {}
        return ObjectMapper().readValue(json, mapType)
like this ?
instead of
        val objectMapper = ObjectMapper()
        val mapType = object : TypeReference<EventSubTopic>() {}
        return ObjectMapper().readValue(json, mapType)

Reply:
No, you need to use the ObjectMapper from DI. Who said anything about TypeReference or that your implementation was wrong

Reply:
@iamareebjamal how i can pass that in type converter because type converter can only have empty constructors

Reply:
and also ListAttendeeIdConverter uses the same pattern because we can't pass anything to type converter

Reply:
@iamareebjamal anything else?

Reply:
You can manually inject the dependency. See field injection

Reply:
but for field injection, we need a context or something type converter don't even have that
i tired using  StandAloneContext.getKoin().koinContext and         val objectMapper: ObjectMapper by inject()

Reply:
OK. Create an issue to convert all object mappers from the one in DI